### ISSUE TITLE
Update Expert List

### ISSUE THREAD

**Issue by DragonmasterJ** [brata-hsdc/brata#95]
I started an expert list in the main brata wiki. I just pulled some information from memory, list needs to be completed. 

I've assigned myself to take the responsibility to make sure it's done. 

@brata-hsdc/brata-devs Does the brata need to be broken into subsections with specific experts for each? Would you be able to help map them if that's the case?

We should also decide whether we want to add the old contributors to the list, or keep to current contributors.

**Comment by DragonmasterJ** [comment]
I just realized that GitHub markdown allows tables. Might be a good idea to utilize that instead of the current bullet list.

**Comment by ellerychan** [comment]
Here is a link to the [Expert List](https://github.com/brata-hsdc/brata/wiki/Expert-List).

**Comment by jawaad-ahmad** [comment]
> Does the brata need to be broken into subsections with specific experts for each?

I think it makes sense to break down as much as makes sense. I added some missing items; still need to fill-in. I also changed the names to links, removed last names, and fixed Dave's user name.

> I just realized that GitHub markdown allows tables. Might be a good idea to utilize that instead of the current bullet list.

The bullets look fine to me for now. Are you expecting more columns?

> We should also decide whether we want to add the old contributors to the list

Old contributors makes sense if we don't have a current POC that can completely fill-in for them. For example, Steve helped a lot with some of the stations, but he has no GitHub user name and the rest of us can fill-in for him because we're familiar enough with his work, so we can leave him off and not bother him.

If there's someone from last year for whom we don't have a replacement, then let's add them in. For example, no one knows last year's Master Server like Dan, so it would be good to keep his name so we know to go to him for questions if we can't answer ourselves.

**Comment by DragonmasterJ** [comment]
> The bullets look fine to me for now. Are you expecting more columns?

No, you are right. It was a thought I had to make it look more clean, but it was before it had any data. It looks fine as is.

Thanks Ellery for putting the link in, I had the feeling I forgot something last night, and you hit the nail on the head.
